firmware/screen/src/ui/screen_pocket_technician.cpp

```
@@ -0,0 +1,450 @@
+#include <stdio.h>
+#include <string.h>
+#include <new>
+
+namespace {
+
+// ---- Module-static cache -------------------------------------------------
+
+constexpr int  PT_MAX_SETUPS    = 64;     // bounded list cache
+constexpr int  PT_MAX_REQUESTS  = 4;      // out-queue depth
+
+struct setup_row_t {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: name, mtime_iso, best_ms

(cppcoreguidelines-pro-type-member-init)